OrbitClientGgp/ClientGgp.h

```diff
@@ -38,6 +38,7 @@ class ClientGgp final : public CaptureListener {
  private:
   ClientGgpOptions options_;
   std::shared_ptr<grpc::Channel> grpc_channel_;
+  std::shared_ptr<Process> target_process;
```

Comment:
please use an `_` at the end of member variables as described in the [style guide](https://google.github.io/styleguide/cppguide.html#Variable_Names).

Reply:
Done, thanks!